### DISCUSSION TITLE
Multi-Page Form

### DISCUSSION THREAD

**Discussion by AlbertBislimi** [solspace/craft-freeform#1389] [Questions & Answers] [answered]
Hello, 

I have a 3 multi page form setup whenever I submit the form only page 1 shows in the email but page 2 and 3 don't show up is there a specific thing I need to do on the admin_notification template I have for the email ?

**Comment by kjmartens** [comment, marked as answer]
Sorry about this @AlbertBislimi,

This was a bug but has now been resolved in Freeform 5.5.2. 🙂
